libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -156,13 +175,28 @@ class MapboxTripNotification constructor(
         }
     }
 
+    /**
+     * Creates [PendingIntent] for open application when notification view is clicked
+     *
+     * @param applicationContext is [Context]
+     * @return [PendingIntent] for open application
+     * @since 1.0.0
+     */
     private fun createPendingOpenIntent(applicationContext: Context): PendingIntent? {
         val pm = applicationContext.packageManager
         val intent = pm.getLaunchIntentForPackage(applicationContext.packageName) ?: return null
         intent.setPackage(null)
         return PendingIntent.getActivity(applicationContext, 0, intent, 0)
     }
 
+    /**
+     * Creates [PendingIntent] for finish navigation process ([TripSession]) when
```

Comment:
This doesn't finish the navigation process per se. We might as well be in a free drive mode, without active guidance and this basically stops the trip session.